### ISSUE TITLE
[Bug] ipv4 address in ipv6 format

### ISSUE THREAD

**Issue by endrepavel** [hestiacp/hestiacp#4599]
### Describe the bug

Can't login to admin panel.

```
# tail  /usr/local/hestia/log/error.log
2024-10-23 10:57:21 v-get-user-salt  'user' '::ffff:145.123.99.99' 'json' [Error 2]
```

### Tell us how to replicate the bug

v-get-user-salt user '::ffff:145.123.99.99' 'json'
Error: invalid IP format :: ::ffff:145.123.99.99

### Which components are affected by this bug?

Control Panel Command Line Interface, Control Panel Web Interface

### Hestia Control Panel Version

1.8.12

### Operating system

Debian 12

### Log capture

```shell
2024-10-23 10:57:21 v-get-user-salt  'user' '::ffff:145.123.99.99' 'json' [Error 2]
```

**Comment by jaapmarcus** [comment]
https://github.com/hestiacp/hestiacp/blob/ceff32c8cff176f9dc1bab3460d1db629df983a9/web/login/index.php#L116-L132

This is part of the code causing the issue 

But both  $_SERVER["REMOTE_ADDR"]; and  $_SERVER["HTTP_CF_CONNECTING_IP"]; should be ipv4 or ipv6  and not a combined mess

**Comment by endrepavel** [comment]
I still have the problem.
HestiaCP is behind a reverse proxy, maybe that matters, but I can't change that.

**Comment by jaapmarcus** [comment]
Fixed #4606
